src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt

```diff
@@ -572,6 +569,8 @@ class CreateMafVcf : CliktCommand(help = "Create g.vcf and h.vcf files from Anch
                         "Checksum=\"Md5\",RefRange=\"${refSeqHash}\">",
                 VCFHeaderVersion.VCF4_2
             )
+        } else {
+           println("convertMetaDataRecordToHVCF: asmHeaders already contains key ${assemblyHaplotypeHash}")
```

Comment:
Should this be added to a logger?

Reply:
Yes - I created a logger for this class and changed all the println's to myLogger values